### PR TITLE
Fix: Do not reference source SvgDocument.

### PR DESCRIPTION
**Pull request by H1Gdev**
#### What does this implement/fix? Explain your changes.

Do not reference source SvgDocument.

### CODE REVIEW

File: Source/Painting/SvgDeferredPaintServer.cs [hunk 1]
```diff
@@ -59,7 +54,6 @@ public override SvgElement DeepCopy()
         public override SvgElement DeepCopy<T>()
         {
             var newObj = base.DeepCopy<T>() as SvgDeferredPaintServer;
-            newObj.Document = this.Document;
```
Comment (H1Gdev) [inline Source/Painting/SvgDeferredPaintServer.cs]: Not copy Document.

File: Source/Painting/SvgDeferredPaintServer.cs [hunk 2]
```diff
@@ -1,45 +1,40 @@
...
-                                              e.Color != SvgColourServer.Inherit && e.Color != SvgColourServer.None
+                                        where e.Color != None && e.Color != SvgColourServer.NotSet &&
+                                              e.Color != SvgColourServer.Inherit
                                         select e).FirstOrDefault();
                     _concreteServer = (colorElement == null ? SvgColourServer.NotSet : colorElement.Color);
                 }
                 else
                 {
-                    _concreteServer = this.Document.IdManager.GetElementById(this.DeferredId) as SvgPaintServer;
+                    _concreteServer = styleOwner.OwnerDocument.IdManager.GetElementById(this.DeferredId) as SvgPaintServer;
```
Comment (H1Gdev) [inline Source/Painting/SvgDeferredPaintServer.cs]: Use styleOwner's OwnerDocument.

File: Source/Painting/SvgDeferredPaintServer.cs [hunk 3]
```diff
@@ -1,45 +1,40 @@
...
     {
         private bool _serverLoaded = false;
         private SvgPaintServer _concreteServer;
 
-        public SvgDocument Document { get; set; }
         public string DeferredId { get; set; }
 
         public SvgDeferredPaintServer() { }
-        public SvgDeferredPaintServer(SvgDocument document, string id)
+        public SvgDeferredPaintServer(string id)
```
Comment (mrbean-bremen) [inline Source/Painting/SvgDeferredPaintServer.cs]: This changes the public API - I'm a bit reluctant to make this kind of changes.
I would refer this to @tebjan to comment/approve.

Comment (H1Gdev) [reply Source/Painting/SvgDeferredPaintServer.cs]: @mrbean-bremen 

Thank you for your review.
I agree.
I will add new constructor.
I will add ‘Obsolete‘  this and not use ‘document‘ parameter.

**Pull request merged**